### PR TITLE
Update README.md with DKMS hints

### PR DESCRIPTION
**Pull request by ray0711**
Add DKMS installation hints. Not exactly beautiful but seems to work.

### CODE REVIEW

**Comment by DanielGibson** [comment]
IIRC you just have to check out the repo (or download and unpack the source) and run `make dkms` as root

**Comment by ray0711** [comment]
hahaha, reading the makefile would have saved me some minutes of my life :)
I updated the PR. Feel free to decline it if you believe it should be obvious or belongs somewhere else.

File: README.md [hunk 1]
```diff
@@ -174,3 +174,10 @@ NOTE: If `gpioinfo` does not return anything, you may need to figure out which (
 - Support variable amount of disk LEDs
 - ~~Create a new led trigger driver so that we can blink disk LEDs individually, the existing `disk-activity` trigger always blinks all LEDs on activity from any disk~~
   - Pray that [[RFC PATCH v3 00/18] Add block device LED trigger](https://lore.kernel.org/linux-leds/20210819025053.222710-1-arequipeno@gmail.com/) by Ian Pilcher lands in the linux kernel
+
+## DKMS
+DKMS installation to enable module auto-build with kernel upgrades.
```
Comment (mafredri) [inline README.md:179]:
```suggestion
## DKMS

DKMS installation to enable module auto-build with kernel upgrades. 
```

File: README.md [hunk 2]
```diff
@@ -174,3 +174,10 @@ NOTE: If `gpioinfo` does not return anything, you may need to figure out which (
 - Support variable amount of disk LEDs
 - ~~Create a new led trigger driver so that we can blink disk LEDs individually, the existing `disk-activity` trigger always blinks all LEDs on activity from any disk~~
   - Pray that [[RFC PATCH v3 00/18] Add block device LED trigger](https://lore.kernel.org/linux-leds/20210819025053.222710-1-arequipeno@gmail.com/) by Ian Pilcher lands in the linux kernel
+
+## DKMS
+DKMS installation to enable module auto-build with kernel upgrades. 
+
+```
+make dkms
```
Comment (mafredri) [inline README.md:182]:
```suggestion
sudo make dkms
```

**Review by mafredri — APPROVED** [review]
Thanks, appreciate you adding the callout to the readme!

**Pull request merged**